README.md

```diff
@@ -189,6 +189,19 @@ Set an alternative locale for Faker used during the scrubbing process.
 
 (default: None, falls back to Django's default locale)
 
+### `SCRUBBER_MAPPING`:
+
+Define a class and a mapper which does not have to live inside the given model. Useful, if you have no control over the
+models code you'd like to scrub.
+
+````python
+SCRUBBER_MAPPING = {
+    "auth.User": "my_app.scrubbers.UserScrubbers",
+}
+````
```

Comment:
Maybe add a small example of a "pure scrubber" here, similar to #41? (maybe a bit shorter)